internal/controller/platform/standalone/nimservice.go

```diff
@@ -176,6 +176,7 @@ func (r *NIMServiceReconciler) reconcileNIMService(ctx context.Context, nimServi
 
 	// Sync serviceaccount
 	err = r.renderAndSyncResource(ctx, nimService, &renderer, &corev1.ServiceAccount{}, func() (client.Object, error) {
+		logger.Info("Rendering serviceaccount", "nimservice", nimService.Name)
```

Comment:
This will be too verbose if we log for every resource. We need to move this to higher log levels.